cmd/repo-updater/repoupdater/server_test.go

```diff
@@ -774,34 +654,33 @@ func TestRepoLookup_syncer(t *testing.T) {
 			HTTPCloneURL: "https://git-codecommit.us-west-1.amazonaws.com/v1/repos/stripe-go",
 			LastModified: &now,
 		},
-	})
-
-	s := Server{
-		Syncer:      &repos.Syncer{},
-		Store:       store,
-		InternalAPI: &internalAPIFake{},
 	}
 
-	t.Run("not found", func(t *testing.T) {
-		have, err := s.repoLookup(ctx, protocol.RepoLookupArgs{Repo: "github.com/a/b"})
-		if err != nil {
-			t.Fatal(err)
-		}
-		want := &protocol.RepoLookupResult{ErrorNotFound: true}
-		if !reflect.DeepEqual(have, want) {
-			t.Error(cmp.Diff(have, want))
-		}
-	})
-
 	testCases := []struct {
-		name         string
-		repoName     string
-		wantRepoInfo *protocol.RepoInfo
+		name                  string
+		args                  protocol.RepoLookupArgs
+		reposInStore          []*repos.Repo
+		want                  *protocol.RepoLookupResult
```

Comment:
When doing multiple have, want comparisons in a single test case, it's confusing to call one of those parameters want.

```suggestion
		result                  *protocol.RepoLookupResult
```